<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>373</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ManipQT</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourses.qrc">
    <normaloff>:/logo/logo/industrial-robot.png</normaloff>:/logo/logo/industrial-robot.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Режим</string>
    </property>
    <widget class="QRadioButton" name="Usual_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Обычный</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="OMP_radioButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>OpenMP</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Calculate_pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Исходные конструктивные данные</string>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>l3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Xb</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Za</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>OA1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>l4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Yb</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>O1A</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>l1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>l2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>O01</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Yc</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Zd</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Y0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Xc</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Φ</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="fi_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.323000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="Xb_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>355</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="Yb_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>755</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="Za_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>750</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="Zd_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>-40</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="Xc_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>-355</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="Yc_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>755</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="Y0_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>755</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="O1A_spinBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>750</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="l4_spinBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>-40</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="O01_spinBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>-40</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="l3_spinBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>1352</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="l2_spinBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>1500</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="l1_spinBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>1400</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="OA1_spinBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-2000</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="value">
      <number>750</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Результаты</string>
    </property>
    <widget class="QTextBrowser" name="Result_textBrowser">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>421</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Начальные координаты захвата Е</string>
    </property>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>α33</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>α0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>a</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>b</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>α23</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>α13</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="alpha0_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-500</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="alpha23_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-500</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="alpha_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-500</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>90</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="alpha13_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-500</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="alpha33_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-500</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="b_spinBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-500</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>220</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="Export_menu">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Экспорт..</string>
     </property>
     <addaction name="actionTXT"/>
     <addaction name="actionXML"/>
    </widget>
    <addaction name="Export_menu"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action_OpenMP"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTXT"/>
   <addaction name="actionXML"/>
   <addaction name="action_OpenMP"/>
   <addaction name="action_about"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTXT">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/txt.png</normaloff>:/img/img/txt.png</iconset>
   </property>
   <property name="text">
    <string>TXT</string>
   </property>
   <property name="toolTip">
    <string>Сохранение результатов в TXT</string>
   </property>
  </action>
  <action name="actionXML">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/xml.png</normaloff>:/img/img/xml.png</iconset>
   </property>
   <property name="text">
    <string>XML</string>
   </property>
   <property name="toolTip">
    <string>Сохранение результатов в XML</string>
   </property>
  </action>
  <action name="action_OpenMP">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/img/img/settings.png</normaloff>:/img/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>Настройки OpenMP..</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/logo/logo/guide.png</normaloff>:/logo/logo/guide.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
